### PR TITLE
Some cleanups for megacheck warnings

### PR DESCRIPTION
**Pull request by knweiss**
Please review (esp the `grpc.Code()` commit).

See the individual commit messages for details.

I've only tested with `go test` so far.

### CODE REVIEW

**Comment by knweiss** [comment]
Travis-CI is broken. See #40.

**Comment by brancz** [comment]
@knweiss you also need to sign the CLA for us to be able to merge these changes. Also even if flaky I'd prefer to see the tests pass.

**Comment by brancz** [comment]
lgtm

File: util.go [hunk 1]
```diff
@@ -37,13 +37,13 @@ func splitMethodName(fullMethodName string) (string, string) {
 }
 
 func typeFromMethodInfo(mInfo *grpc.MethodInfo) grpcType {
-	if mInfo.IsClientStream == false && mInfo.IsServerStream == false {
+	if !mInfo.IsClientStream && !mInfo.IsServerStream {
```
Comment (Bplotka) [inline util.go]: lol (:

**Pull request merged**